service/src/main/java/uk/gov/hmcts/reform/fpl/model/CaseData.java

```diff
@@ -46,7 +47,7 @@
     private final List<@NotNull(message = "You need to add details to applicant")
         Element<Applicant>> applicants;
     @NotNull(message = "You need to add details to respondents")
-    private final List<@NotNull(message = "You need to add details to respondents")Element<Respondent>> respondents1;
+    private final List<@NotNull(message = "You need to add details to respondents") Element<Respondent>> respondents1;
 
     @Valid
     private final Respondent getFirstRespondent() {
```

Comment:
do we use this method?

Reply:
no - auto reformat must have added the space to it, as it should